src/stores/ProjectUsersStore.js

```diff
@@ -19,22 +19,19 @@ export const useProjectUsersStore = defineStore({
       this.users = response.data.users
       this.isLoaded = true
     },
-    async editUser(projectId, linkId, user) {
+    async editUser(projectId, linkId, user, groupsJoined) {
       const url = `${
         import.meta.env.VITE_API_URL
       }/projects/${projectId}/users/${linkId}/edit`
-      const response = await axios.post(url, { user })
+      const response = await axios.post(url, { user, groupsJoined })
       if (response.status == 200) {
         const user = response.data.user
-        this.updateUser(linkId, user)
+        const index = this.removeUserByLinkId(user.link_id)
+        this.users.splice(index, 0, user)
```

Comment:
Got it. I did it this way because I wanted all the groups to be ordered alphabetically when a change was made, should I still try to achieve that but in a different way?